chef-config/lib/chef-config/workstation_config_loader.rb

```diff
@@ -265,7 +265,7 @@ def find_default_key(key_names)
 
     def highlight_config_error(file, line)
       config_file_lines = []
-      IO.readlines(file).each_with_index { |l, i| config_file_lines << "#{(i + 1).to_s.rjust(3)}: #{l.chomp}" }
+      IO.readlines(file).each_with_index do |l, i| config_file_lines << "#{(i + 1).to_s.rjust(3)}: #{l.chomp}" end
```

Comment:
this should be an `each_with_object` or `inject`, so i guess this is right and points out the code is ugly for other reasons?

Reply:
This screams `map` to me.